docs/docs/recipes/manage-users/you-dont-need-a-user-table.md

```diff
@@ -1,39 +1,37 @@
 ---
-sidebar_position: 3
+sidebar_position: 4
 ---
 
-# You don't need a user table
+# Common practice: you don't need a user table 
 
 ## Identify user
 
-We recommend using `user.id` to identify a user since we guarantee every user has a unique and non-null `id` property.
+We suggest utilizing the `user.id` property to identify a user, as we ensure each user has a distinct and non-null `id`. Additionally, the `username`, `primary_email`, and `primary_phone` properties are unique but may be empty. Therefore, it is important to handle `null` values appropriately if you intend to use these properties as identifiers for a user.
 
-Also, `username` `primary_email` and `primary_phone` are unique properties. But they may be empty. Remember to handle `null` properly if you want to use these properties to identify a user.
+## Why is there no user table
```

Comment:
A little bit misleading. Something like, why you don't need a separate user table

Reply:
Updated